packages/react-hooks/src/useThresholds.ts

```diff
@@ -3,7 +3,11 @@
 
 import { useMemo } from 'react';
 
-import { PROPOSE_THRESHOLDS, REJECT_THRESHOLDS, SLASH_THRESHOLDS, TREASURY_THRESHOLDS } from '@polkadot/apps-config';
+import { PROPOSE_THRESHOLDS,
+  REJECT_THRESHOLDS,
+  SLASH_THRESHOLDS,
+  Threshold,
+  TREASURY_THRESHOLDS } from '@polkadot/apps-config';
```

Comment:
```suggestion
import { PROPOSE_THRESHOLDS, REJECT_THRESHOLDS, SLASH_THRESHOLDS, Threshold, TREASURY_THRESHOLDS } from '@polkadot/apps-config';
```

Additionally, Threshold is a type import, should be explicitly one via `import type`